### ISSUE TITLE
Failed to start the application if signed in to Dockerhub

### ISSUE THREAD

**Issue by SteffenGottschalk** [thenativeweb/wolkenkit#133]
### What is this bug about?

If I am logged in to Dockerhub with my private docker id on Windows, I was not able to start my wolkenkit application. The CLI did not give any hints about the cause of the error. So I ran the `start` command again with the `--verbose` flag. This caused the following output…

<img width="1139" alt="docker" src="https://user-images.githubusercontent.com/1318209/53283992-1a50f200-374e-11e9-9363-39a41b5cf6f4.PNG">

### What steps are needed to reproduce the bug?

- Sign in to Dockerhub on Windows.
- Start an application.

### What is the expected result?

- The application should start.
- If the application fails, it should tell me the cause of the failure.

### What is the actual result?

- The start of the application fails.
- It does not give any additional details about the cause of the failure.

### What else should we know?

- After logging out from docker, the `start`command worked.

**Comment by goloroden** [comment]
Hey @SteffenGottschalk 😊 

Thanks for submitting this issue.

I have one question, though: Is the expected result actually to get the error message without providing the `--verbose` flag, or is the expected result rather that the application starts, even if you are logged in?

> Get the error message without starting 'wolkenkit --verbose'.

says the first one, but why would you expect this? But maybe I didn't get your scenario anyway 😉. You  wrote:

> If i am logged in to Dockerhub with my privet docker id […]

What do you mean with a *privat docker id*? Just being logged in to the Docker Hub with your credentials? Or being logged in to a special private part of it? Or …?

**Comment by mattwagl** [comment]
I talked to @SteffenGottschalk and updated the text of this issue accordingly. I'll try to reproduce the issue on a windows hardware this week when I'm back at the office. Besides the main issue of failing during the start on windows, the second problem here is that as a developer you are not able to see the cause of an error. You have to run the command again in verbose mode. We had this problem a few times already. It's especially frustrating if you are not able to reproduce an error, since you're just left with a failing command. So I'd really vote for always displaying the error details if one occurs. If that sound's reasonable, we could open another issue for this topic.

**Comment by SteffenGottschalk** [comment]
We've setup the project on another windows machine (Windows 10) and got the same error while being logged into Docker hub. This is the full verbose output of the `start` command:

```shell
  Starting the application...
  Verifying health on environment default...
  Application host local.wolkenkit.io resolves to 127.0.0.1.
  Docker server resolves to 127.0.0.1 and ::1.
  Verifying application status...
  Verifying that ports are available...
  Verified that port 3000 is available.
  Verified that port 3001 is available.
  Verified that port 3030 is available.
  Installing wolkenkit 3.1.0 on environment default...
  Pulling thenativeweb/wolkenkit-broker:3.1.0...
  Pulling thenativeweb/wolkenkit-core:3.1.0...
  Pulling thenativeweb/wolkenkit-depot:3.1.0...
  Pulling thenativeweb/wolkenkit-flows:3.1.0...
  Pulling thenativeweb/wolkenkit-mongodb:3.1.0...
  Pulling thenativeweb/wolkenkit-node-modules:3.1.0...
  Pulling thenativeweb/wolkenkit-postgres:3.1.0...
  Pulling thenativeweb/wolkenkit-proxy:3.1.0...
  Pulling thenativeweb/wolkenkit-rabbitmq:3.1.0...
  Failed to pull thenativeweb/wolkenkit-depot:3.1.0.
✗ Failed to start the application.
  Error response from daemon: Get https://registry-1.docker.io/v2/thenativeweb/wolkenkit-depot/manifests/3.1.0: unauthorized: incorrect username or password

  ExecutableFailed: Error response from daemon: Get https://registry-1.docker.io/v2/thenativeweb/wolkenkit-depot/manifests/3.1.0: unauthorized: incorrect username or password

    at C:\Users\...\node_modules\wolkenkit\dist\shell\exec.js:26:18
    at ChildProcess.exithandler (child_process.js:301:5)
    at ChildProcess.emit (events.js:189:13)
    at maybeClose (internal/child_process.js:970:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
```

**Comment by mattwagl** [comment]
We tried different accounts and finally found the root cause. The problem here is that you have to log into Docker using your username, not your e-mail. If you use your e-mail to login, you will always get the error `unauthorized: incorrect username or password` error, even when pulling other public images. [See this an open issue in the docker-feedback](https://github.com/docker/hub-feedback/issues/935) repo for more details on this.

I'm goint to close this issue now but will open another one for always displaying the error details when a command failed, as this will hopefully help us to track down issues like this in the future much faster.
